evap/rewards/tools.py

```diff
@@ -25,11 +26,19 @@
 
 @login_required
 @transaction.atomic
-def save_redemptions(request, redemptions: Dict[int, int]):
+def save_redemptions(request, redemptions: Dict[int, int], previous_reward_points):
     # lock these rows to prevent race conditions
     list(request.user.reward_point_grantings.select_for_update())
     list(request.user.reward_point_redemptions.select_for_update())
 
+    # check consistent previous redeemed points
```

Comment:
Should we extend this comment to explain why we are _not_ also comparing rewarded points? What do you think @He3lixxx ?

Reply:
I'm fine with both the current version and an additional comment.